app/controllers/AssessmentController.scala

```diff
@@ -63,8 +63,8 @@ class AssessmentController @Inject() (
             .get(CategorisationDetailsQuery(recordId))
             .flatMap { categorisationInfo =>
               categorisationInfo.getAssessmentFromIndex(index).map { assessment =>
-                val listItems = assessment.getExemptionListItems
-                val form      = formProvider(listItems.size)
+                val codesAndDescriptions = assessment.getCodesZippedWithDescriptions
+                val form                 = formProvider()
 
                 val preparedForm = request.userAnswers.get(AssessmentPage(recordId, index)) match {
```

Comment:
we can use the generic prepared form method in the BaseController trait 

`val preparedForm = prepareForm(AssessmentPage(recordId, index), form)` 
you might need to merge the current main.

Reply:
I'll merge main shortly and then notify in slack!